integration-tests/src/test/java/oracle/kubernetes/operator/JrfInOperatorTest.java

```diff
@@ -403,6 +412,263 @@ public void testAutoAndCustomSitConfigOverrides() throws Exception {
     logger.info("SUCCESS - " + testMethod);
   }
 
+  /**
+   * test the Rolling restart behavior in the jrf domain cluster level currently there are two bugs
+   * 29678557, 29720185, the test will fail
```

Comment:
Please describe what the test is doing instead of referring to internal bugdb numbers since users outside of Oracle don't have access to bugdb.